wordpress-mobile/WordPress-Android
# Issue/connect all bp instances to store

This PR connects following prompt entry points to the store:
- Compact card
- Onboarding dialog
- Reminder notification

#16504 is perquisite and #16524 is recommended for testing

To test:
- Enabled BP flag and restart the app.
- Press + button in My Site screen and confirm that the compact card is visible, and tapping on it opens editor and fills it with correct content*.
- From onboarding dialog press Try It Now and confirm that it opens editor and fills it with correct content*
- Confirm that reminder notification has a correct prompt, and tapping on it opens editor and fills it with correct content*

*if #16524 is merged

## Regression Notes
1. Potential unintended areas of impact
- Some current prompt functionality broke when transitioning to live prompts.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Manual testing, updated existing tests.

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
